std/outbuffer.d

```diff
@@ -202,6 +202,14 @@ class OutBuffer
             fill0(alignsize - nbytes);
     }
 
+    /*********************************
+     * Clear the data in the outbuffer.
+     */
```

Comment:
FYI: `outbuffer` is a very old module, the more modern equivalent docstring would `/// Clear ...` ;-)